src/components/pages/Dashboard/index.js

```diff
@@ -1,22 +1,43 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import '../../../styles/InstructorStyles/index.less';
-import { Layout } from 'antd';
-import InstructorSidebar from '../InstructorHome/InstructorSidebar';
-import InstructorStats from './InstructorStats';
+import { useOktaAuth } from '@okta/okta-react';
+import { getCurrentUser } from '../../../redux/actions/userActions';
+import AdminHome from '../AdminHome/index';
+import InstructorHome from '../InstructorHome/index';
+import ParentHome from '../ParentHome/ParentHome';
 
-const { Content } = Layout;
-const InstructorDashboard = () => {
-  return (
-    <div>
-      <Layout>
-        <InstructorSidebar />
-        <Content>
-          <InstructorStats />
-          {/* Other Dashboard Components */}
-        </Content>
-      </Layout>
-    </div>
-  );
+import { connect, useDispatch } from 'react-redux';
+
+const InstructorDashboard = props => {
```

Comment:
We should rename this "Dashboard" to match the folder and import in our `index.js`!

Reply:
Alright just pushed these changes